pkg/openstack/client/types.go

```diff
@@ -112,6 +112,9 @@ type Networking interface {
 	GetSecurityGroupbyName(name string) ([]groups.SecGroup, error)
 	// Security Group rules
 	CreateRule(createOpts rules.CreateOpts) (*rules.SecGroupRule, error)
+	ListRules(listOpts rules.ListOpts) ([]rules.SecGroupRule, error)
+	GetRulebyName(name string) ([]rules.SecGroupRule, error)
```

Comment:
should be renamed to `GetRuleByName`. But as said in one of my other comments, I doubt this function is needed at all and should be removed.

Reply:
close as the new solution provider need this